features/step_definitions/work_steps.rb

```diff
@@ -33,30 +33,45 @@
 #
 # If you add to this regexp, you probably want to update all the 
 # similar regexps in the I post/Given the draft/the work steps below.
-When /^I set up (?:a|the) draft "([^\"]*)"(?: with fandom "([^\"]*)")?(?: with freeform "([^\"]*)")?(?: with category "([^\"]*)")?(?: (?:in|to|with) (?:the )?collection "([^\"]*)")?(?: as a gift (?:for|to) "([^\"]*)")?$/ do |title, fandom, freeform, category, collection, recipient|
+When /^I set up (?:a|the) draft "([^\"]*)"(?: with fandom "([^\"]*)")?(?: with freeform "([^\"]*)")?(?: with second freeform "([^\"]*)")?(?: with category "([^\"]*)")?(?: (?:in|to|with) (?:the )?collection "([^\"]*)")?(?: as a gift (?:for|to) "([^\"]*)")?(?: as part of a series "([^\"]*)" by "([^\"]*)")?$/ do |title, fandom, freeform, freeform2, category, collection, recipient, series, series_user|
```

Comment:
The problem is that this is being used in a javascript test and the test and the web ui run in different threads so you can't access current_user